R/autoplot.R

```diff
@@ -20,8 +19,8 @@
 #' @aliases autoplot
 #'
 #' @examples
-#' # Create an example aedseo_tsd object
-#' aedseo_tsd_object <- tsd(
+#' # Create an example tsd object
+#' tsd_object <- tsd(
```

Comment:
I realise that you are just renaming the old code, but I think something like "time_series" is more readable in an example than "tsd_object"